### DISCUSSION TITLE
Curso FullStack com Visual Studio

### DISCUSSION THREAD

**Discussion by 1983w** [balta-io/forum#574] [Tira Dúvidas] [answered]
boa tarde, existe alguma maneira de acompanhar o curso utilizando o visual studio? 
Talvez abrir o terminal no Visual Studio e executar os mesmos comandos.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @1983w, como vai?
Existe sim, pode acompanhar normalmente pelo Visual Studio ou Visual Studio Code.

No VS Code é só apertar `CTRL + `` e no Visual Studio é só ir em `View > Terminal`
